Automattic/newspack-theme
# Update theme/folder structure for move to parent/child themes -- attempt 2

### All Submissions:






### Changes proposed in this Pull Request:

This is a re-do of #630, but simplified to remove the multiple rebase commits to make it easier to follow.

It also includes a fix for the issue described in the previous PR, where I made a mistake in the webpack updates, so not all of the JS was being added to the /dist directory (whoops!). 

Related #594.

### How to test the changes in this Pull Request:

Largely copied over from #630:

1. Clone the repository directly into the `/themes` folder using `git clone git@github.com:Automattic/newspack-theme.git .`. You will need to empty the directory first; if you still get the error `fatal: destination path '.' already exists and is not an empty directory.`, you can check for hidden files using `ls -a`.
Alternatively, you can clone the repo into a location separate from your themes directory, and symlink the nested 'newspack-theme' directory to a wp-content/themes directory on your test site. 
2. Apply the PR.
3. In the root of the directory, run `npm install && composer install`.
4. In the same location, try running `npm run build` and `npm run release:archive`. 
5. Confirm the theme is running as expected on your site.
6. Confirm that a zip has been generated in the `/releases` directory. 
7. Try switching to one of the languages with a translation (eg Spanish); confirm that the strings on the front-end of the theme (like 'by' on a single post), and strings in the editor (like the 'Large' and 'Small' options for the Featured Image settings) are translated. 

### Other information:
